src/Symfony/Component/Yaml/Parser.php

```diff
@@ -730,15 +730,15 @@ private function cleanup($value)
         $this->offset += $count;
 
         // remove leading comments
-        $trimmedValue = preg_replace('#^(\#.*?\n)+#s', '', $value, -1, $count);
+        $trimmedValue = preg_replace('#^(\n+)?(\#.*?\n)+#s', '', $value, -1, $count);
```

Comment:
My regex skills are not top-notch, but what if there's spaces after the new lines and before the next new line?